modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/defender.dm

```diff
@@ -0,0 +1,146 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/defender
+	name = "alien defender"
+	caste = "defender"
+	maxHealth = 300
+	health = 300
+	icon_state = "aliendefender"
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	var/datum/action/cooldown/mob_cooldown/charge/basic_charge/defender/charge
+	var/datum/action/cooldown/spell/aoe/repulse/xeno/crushing/tail_sweep
+	/// var/datum/action/cooldown/alien/skyrat/crest_defence/crest_defending
+	next_evolution = /mob/living/carbon/alien/humanoid/skyrat/warrior
```

Comment:
just warrior, no bull?

Reply:
If you want to know why, I want you to go look at tgmc code and look at what they do for making crusher/bull charging work, 300 lines later you will understand why I didn't